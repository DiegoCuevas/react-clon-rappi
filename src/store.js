import { createStore, applyMiddleware, compose } from "redux";
import reducer from "./reducer";
import thunk from "redux-thunk";
import logger from "redux-logger";

const composer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(reducer, composer(applyMiddleware(thunk, logger)));

export default store;
