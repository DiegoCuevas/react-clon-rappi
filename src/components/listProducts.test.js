import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";

import store from "../store";
import ListProducts from "./listProducts";

test("listProducts component", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <ListProducts />
    </Provider>
  );

  expect(asFragment()).toMatchSnapshot();
});
