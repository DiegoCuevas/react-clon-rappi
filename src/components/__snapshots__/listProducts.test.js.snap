// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listProducts component 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<section
    class="emotion-0"
  />
</DocumentFragment>
`;
